javascript/javascript3/week3/homework/script.js

```diff
@@ -0,0 +1,89 @@
+const BASE_URL = `https://crudcrud.com/api/${API_KEY}`;
+const inputUrl = document.querySelector('.input-url');
+const listAllScreenshots = document.querySelector('.list-all-screenshots');
+const renderListOfScreenshots = document.querySelector('.render-list-of-screenshots');
+const renderScreenshot = document.querySelector('.render-screenshot');
+let screenshotUrlResponse = [];
+const sessionId = sessionStorage.getItem('id');
+
+const inputScreenshot = async () => {
```

Comment:
I would suggest a naming that explains a bit more what the function does. For example: getWebsiteScreenshot.